projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="messageEvents$ | async as messageEvents">
-  <div class="container">
+  <div class="container" role="list">
```

Comment:
The attribute role="list" has no effect on this div as the message-card divs are lower in the hierarchy. The attribute role="list" should be moved to the div with id="cx-messages".